### ISSUE TITLE
Cannot read property 'setProperty' of undefined

### ISSUE THREAD

**Issue by Tomekmularczyk** [andreypopp/react-textarea-autosize#196]
I've upgraded package from 6.0.1 to version 6.0.2 and I'm using node v9.7.1

I get this error while I do server-side rendering:

```
TypeError: Cannot read property 'setProperty' of undefined

at /Users/praca/Desktop/Projekty/frp/node_modules/react-textarea-autosize/dist/react-textarea-autosize.cjs.js:93
    node.style.setProperty(key, HIDDEN_TEXTAREA_STYLE[key], 'important');
```

It doesn't happen when I run webpack-dev-server. I do not transpile node_modules on server side rendering maybe that's why?

**Comment by Andarist** [comment]
@Tomekmularczyk I've introduced SSR regression while fixing other issue this weekend. Sorry for that. Should be fixed now with 6.0.3. Let me know if you encounter any problems.

**Comment by Tomekmularczyk** [comment]
@Andarist Thanks, I no longer have this problem with 6.0.3.
